web/client/utils/__tests__/CoordinatesUtils-test.js

```diff
@@ -766,6 +766,18 @@ describe('CoordinatesUtils', () => {
         const uc = [2, 4];
         expect(makeBboxFromOWS(lc, uc)).toEqual([2, 2, 4, 4]);
     });
+    it('getCesiumBoundFromOWS passing extent', ()=>{
+        const extent = [1, 1, 5, 5];
+        expect(getCesiumBoundFromOWS(extent)).toEqual({
+            minx: 1, miny: 1, maxx: 5, maxy: 5
+        });
+    });
+    it('getCesiumBoundFromOWS passing extent', ()=>{
```

Comment:
change title of the test
```suggestion
    it('with no extent passed', ()=>{
```